eventmesh-operator/config/samples/eventmesh_v1_runtime.yaml

```
@@ -271,6 +284,14 @@ spec:
               name: runtime-config
           # imagePullPolicy is the image pull policy
           imagePullPolicy: Always
+          # resources describes the compute resource requirements and limits
+          resources:
+            requests:
+              memory: "1024Mi"
+              cpu: "250m"
+            limits:
+              memory: "2048Mi"
+              cpu: "500m"
```

Are these resources set for EventMesh Runtime?

The memory limit set for EventMesh Runtime in start.sh is 8G. Although it typically requires less, the limit of 2G is too strict.

The threadPools in EventMesh Runtime are designed for 4~8 core machines to speed up async processor.

We can use eventmesh image under apache namespace in future updates.

Besides, why are the values of the properties enclosed in double quotation marks here, while they are not in https://github.com/apache/eventmesh/pull/4727?

---

> Are these resources set for EventMesh Runtime?
> 
> The memory limit set for EventMesh Runtime in start.sh is 8G. Although it typically requires less, the limit of 2G is too strict.
> 
> The threadPools in EventMesh Runtime are designed for 4~8 core machines to speed up async processor.
> 
> We can use eventmesh image under apache namespace in future updates.
> 
> Besides, why are the values of the properties enclosed in double quotation marks here, while they are not in #4727? 

1.This side I use their own computer to do the test, due to limited resources, so set some low, easy to do the test, in your opinion, you think runtime, connector, how much more appropriate ? 
2.For this double quote, this is only a format of yaml, this part is a common string, so it can be added or not, the difference is that if there are special symbols in the string, you need to add double quotes. If you need to unify, you can remove this double quotation mark.

---

For a standard deployment senario, this will be enough:

```suggestion
          resources:
            requests:
              memory: 2048Mi
              cpu: 4000m
            limits:
              memory: 8192Mi
              cpu: 8000m
```

---

@Pil0tXia Modifications have been made.